pkg/controller/common/constants.go

```diff
@@ -0,0 +1,18 @@
+package common
+
+import (
+	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
+)
+
+var ComplianceOperatorNamespace = "openshift-compliance"
+
+func init() {
+	ns, err := k8sutil.GetOperatorNamespace()
+	if err == nil {
+		ComplianceOperatorNamespace = ns
+	}
+}
+
+func GetComplianceOperatorNamespace() string {
+	return ComplianceOperatorNamespace
+}
```

Comment:
if we have a function that gets this variable. Why is the variable exported? Let's make it private to this package and force folks t use the function instead.

Reply:
done.